app/services/bufferizer/satellite_tweet.rb

```diff
@@ -0,0 +1,36 @@
+module Bufferizer
+  class SatelliteTweet
+    def self.call(article, tweet, admin_id = nil)
+      return unless article && tweet
+
+      article.tags.find_each do |tag|
+        next if tag.buffer_profile_id_code.blank?
+
+        text = twitter_buffer_text(tweet, article)
+
+        if text.length < 250 && SiteConfig.twitter_hashtag
```

Comment:
Same question here, can the 250 be moved into a constant?

Reply:
I'm not entirely sure where `250` comes from 🤔  but I gave it a shot!